frontend/src/lib/candidate/components/textArea/MultilangTextInput.svelte

```diff
@@ -122,7 +122,7 @@ Input field variant
 
       {#if translationsShown}
         {#each $locales.filter((locale) => locale !== $currentLocale) as locale}
-          <Field>
+          <Field id={translationsShown ? id + '-' + $currentLocale : id}>
             {#if compact}
               <InputField
                 id={id + '-' + locale}
```

Comment:
While we're at it, could you convert all of these constructions to the format `id="{id}-{locale}"`